### DISCUSSION TITLE
Vote to promote @tisvictress to maintainer

### DISCUSSION THREAD

**Discussion by TisVictress** [paketo-buildpacks/feedback#98] [Content Team]
Hey, my name is Victoria and I've been working on the Paketo website initiative since August. I'm currently working on docs automation and I'd love to be promoted to maintainer.

@paketo-buildpacks/content-maintainers 👍 /👎 ?

**Comment by ryanmoran** [comment]
With a super-majority of maintainers voting in the affirmative, congrats @TisVictress on your new maintainership!
